docs/pages/blog/2020-developer-survey-results.md

```diff
@@ -0,0 +1,314 @@
+  <tr><td>1</td><td>styles - keep jss</td><tr>
+  <tr><td>1</td><td>styles - remove JSS</td><tr>
+  <tr><th>25</th><th> typescript</th><tr>
+  <tr><td>19</td><td>typescript - ?</td><tr>
+  <tr><td>3</td><td>typescript - docs</td><tr>
+  <tr><td>3</td><td>typescript - faster check</td><tr>
+  <tr><th>21</th><th> date picker - improve</th><tr>
+  <tr><th>19</th><th> react native</th><tr>
+  <tr><th>13</th><th> form</th><tr>
+  <tr><th>13</th><th> material design updates</th><tr>
+  <tr><th>12</th><th> animations</th><tr>
+  <tr><td>8</td><td>animations - ?</td><tr>
+  <tr><td>2</td><td>animations - docs</td><tr>
+  <tr><td>1</td><td>animations - declarative API</td><tr>
+  <tr><td>1</td><td>animations - SVG</td><tr>
+  <tr><th>11</th><th> test</th><tr>
+  <tr><td>4</td><td>test - jest</td><tr>
+  <tr><td>3</td><td>test - stable snapshot</td><tr>
+  <tr><td>1</td><td>test - ?</td><tr>
+  <tr><td>1</td><td>test - docs</td><tr>
+  <tr><td>1</td><td>test - styles</td><tr>
+  <tr><td>1</td><td>test - testing library integration</td><tr>
+  <tr><th>8</th><th> more opinionated</th><tr>
+  <tr><th>8</th><th> simplify</th><tr>
+  <tr><th>7</th><th> class components</th><tr>
+  <tr><th>7</th><th> mobile</th><tr>
+  <tr><th>7</th><th> fewer breaking changes</th><tr>
+  <tr><th>7</th><th> designers</th><tr>
+  <tr><td>2</td><td>designers - bridge design tools and code</td><tr>
+  <tr><td>2</td><td>designers - Adobe XD material</td><tr>
+  <tr><td>2</td><td>designers - Figma material</td><tr>
+  <tr><td>1</td><td>designers - ?</td><tr>
+  <tr><th>6</th><th> free vs paid balance</th><tr>
+  <tr><th>6</th><th> tree view - improve</th><tr>
+  <tr><th>6</th><th> less abstracted components</th><tr>
+  <tr><th>5</th><th> more abstracted components</th><tr>
+  <tr><th>4</th><th> accessibility</th><tr>
+  <tr><td>1</td><td>accessibility - ?</td><tr>
+  <tr><td>1</td><td>accessibility - auto id</td><tr>
+  <tr><td>1</td><td>accessibility - full audit</td><tr>
+  <tr><td>1</td><td>accessibility - more examples</td><tr>
+  <tr><th>4</th><th> system</th><tr>
+  <tr><td>1</td><td>system - CSS grid</td><tr>
+  <tr><td>1</td><td>system - docs</td><tr>
+  <tr><td>1</td><td>system - performance</td><tr>
+  <tr><td>1</td><td>system - rework breakpoints</td><tr>
+  <tr><th>3</th><th> components consistency</th><tr>
+  <tr><th>3</th><th> grid - improve</th><tr>
+</table>
+
+### Comparison with last year
+
+There are a couple of noticeable differences compared to last year.
+Some can be explained by our work, others by an evolution of the ecosystem.
+Each item is prefixed by the multiplication factor of the pain point for 2020 relative to 2019.
+
+Decreasing pain:
+- x0: Slider. No requests. The requirements are mostly met, especially with the introduction of the range feature.
+- x0.1: Strict mode. We did a lot for it this year. However, since Create React App has made this mode a default, we have seen a lot more requests for it.
```

Comment:
```suggestion
- x0.1: Strict mode. We fixed a lot of strict mode compatibility issues this year. However, since Create React App has made this mode a default, we have seen a lot more requests for it.
```